handsontable/src/plugins/mergeCells/mergeCells.js

```diff
@@ -152,6 +153,7 @@ export class MergeCells extends BasePlugin {
         return false;
       }
     });
+    this.addHook('beforePaste', (...args) => this.onBeforePaste(...args));
```

Comment:
```suggestion
    this.addHook('beforePaste', (...args) => this.#onBeforePaste(...args));
```
It'd be nice to have this defined as private method.

Reply:
It has been changed within https://github.com/handsontable/handsontable/pull/10444/commits/7155bbfc8c9953edcce21bd725c44f6539fd8de5.